doc/minutes/2022-12-01_first_meeting.md

```diff
@@ -0,0 +1,73 @@
+---
+title:  "First meeting YOS Award"
+date:   "30-11-2022"
+editor_options:
+  markdown:
+    canonical: false
+---
+
+**Attendees:** <names and surnames of attendees>
+
+-   Daniel Morillo Cuadrado
+
+-   Roberto Mediavilla Torres
+
+-   Cristina Rodríguez-Prada
+
+-   Lea Francia
+
+-   Jesus Sierralaya
+
+**Scribe:** Jesus Sierralaya
+
+**Schedule:** 11:00 - 11:30 <!-- # Init time - End time -->
+
+# Action points
+
+## Project Proposals
+
+### Robert's proposal
+
+-   Cloud data management system and federated analysis
+
+-   Software Obiba Opal
+
+-   We already have a physical server
+
+### Lea's proposal
+
+-   Conducting a study from scratch to publication with GitHub
+
+### Daniel's proposal
+
+-   Similar to Lea's proposal
+
+-   We can use Synchros as expample
+
+#### Debate <!-- # Delete if unneeded -->
+
+-   What we're going to do with the money?
+
+    -   Pay a freelancer to maintain the server
+
+    -   General communication
+
+    -   Staff training
+
+#### Agreements <!-- # Delete if unneeded; delete/repeat each point as needed -->
+
+-   **Approved**:
+
+-   Make a combination of both proposals
+
+-   Use the money for staff training
+
+# **Task summary:**
```

Comment:
Tasks in the "task summary" should also be within the action points. This here is a summary of the tasks previously stated in the action points.